### ISSUE TITLE
change the second example code

### ISSUE THREAD

**Issue by aborruso** [eddelbuettel/littler#80]
Hi,
if I run the second example 

```
ls -l /boot | awk 'BEGIN {print "size"} !/^total/ {print $5}' | r -de "print(summary(X[,1])); stem(X[,1])"
```

I have 

```
   Mode
logical
Error in stem(X[, 1]) : 'x' must be numeric
```

because my `/boot` folder is empty. Probably it should be better to change `/boot` in `/bin`, to use a more generic path.

Thank you

**Comment by eddelbuettel** [comment]
Howdy. I always use `/boot` because the size difference there is so big that it makes for a good example.  `/usr/sbin` may work too.  In any event, "just an example"  :)

What type of system are you on with `/boot` empty?

**Comment by aborruso** [comment]
Hi @eddelbuettel and thank you.

I'm using debian (inside Windows Subsystem for Linux).

I know that probably this is not an important thing, but today I have started my first littler test:

- the first example does not work (for me), and I had to find out that using a different shell I must change the command;
- the second example does not work, because I have an empty folder, and I had to find out what's the problem.

For a newby as me was a little stunned.

Thank you again
